packages/databyss-ui/modules/Sidebar/Sidebar.js

```diff
@@ -116,23 +108,24 @@ const Sidebar = () => {
           alignItems="center"
         >
           {/* header */}
-          <Header onHeaderClick={onHeaderClick} menuItem={menuItem} />
+          <Header onHeaderClick={onHeaderClick} />
           {/* content */}
           {menuItem !== 'search' && <Separator color="border.1" />}
-          <Search menuItem={menuItem} onClick={() => setMenuItem('search')} />
+          <Search
+            onClick={() => {
+              navigateSidebar('/search')
+            }}
+          />
           <SidebarContent />
         </List>
         {/* footer  */}
         <View position="absolute" bottom={0} left={0} width="300px">
-          <Footer setMenuItem={setMenuItem} />
+          <Footer />
         </View>
       </View>
     </View>
   ) : (
-    <SidebarCollapsed
-      onToggleMenuOpen={onToggleMenuOpen}
-      setMenuItem={setMenuItem}
-    />
+    <SidebarCollapsed onToggleMenuOpen={onToggleMenuOpen} />
```

Comment:
Move `menuOpen` state to `NavigationContext.state.sidebarExpanded` so we don't need to pass props